locale/de_DE/locale.po

```diff
@@ -74,10 +77,50 @@ msgstr "Primäres Format für bibliographische Angaben"
 msgid "plugins.generic.citationStyleLanguage.settings.citationFormatsPrimaryDescription"
 msgstr "Wählen Sie aus, in welchem Format Sie bibliografische Angaben standardmäßig auf der Artikelseite anzeigen möchten."
 
+msgid "plugins.generic.citationStyleLanguage.settings.citationFormatsPrimaryDescription.omp"
+msgstr "Wählen Sie aus, in welchem Format Sie bibliografische Angaben standardmäßig auf der Buchseite anzeigen möchten."
+
 msgid "plugins.generic.citationStyleLanguage.settings.publisherLocation.description"
 msgstr ""
 "Einige Zitierstile erfordern die Angabe des geografischen Standortes des "
 "Verlags, wie zum Beispiel \"London, U.K.\"."
 
 msgid "plugins.generic.citationStyleLanguage.settings.publisherLocation"
 msgstr "Standort des Verlags"
+
+msgid "plugins.generic.citationStyleLanguage.settings.citationUserGroups"
+msgstr "Autoren und Übersetzer"
+
+msgid "plugins.generic.citationStyleLanguage.settings.citationUserGroupsDescription"
+msgstr "In manchen Fällen gibt es neben den Autoren noch Übersetzer. Wählen Sie bitte je eine Rolle für Autoren und Übersetzer."
+
+msgid "plugins.generic.citationStyleLanguage.settings.citationUserGroups.omp"
+msgstr "Autoren, Herausgeber und Übersetzer"
+
+msgid "plugins.generic.citationStyleLanguage.settings.citationUserGroupsDescription.omp"
+msgstr "In manchen Fällen gibt es neben den Autoren noch Herausgeber oder Übersetzer. Wählen Sie bitte je eine Rolle für Autoren, Herausgeber und Übersetzer."
+
+msgid "plugins.generic.citationStyleLanguage.settings.citationChooseAuthor"
+msgstr "Autor:"
+
+msgid "plugins.generic.citationStyleLanguage.settings.citationChooseEditor"
+msgstr "Herausgeber:"
+
+msgid "plugins.generic.citationStyleLanguage.settings.citationChooseTranslator"
+msgstr "Übersetzer:"
+
+msgid "plugins.generic.citationStyleLanguage.settings.citationChooseChapterAuthor"
+msgstr "Kapitelautor:"
+
+msgid "plugins.generic.citationStyleLanguage.settings.citationOptionChooseAuthor"
+msgstr "Bitte wählen Sie eine Rolle für die Autoren"
+
+msgid "plugins.generic.citationStyleLanguage.settings.citationOptionChooseEditor"
+msgstr "Bitte wählen Sie eine Rolle für die Herausgeber"
+
+msgid "plugins.generic.citationStyleLanguage.settings.citationOptionChooseTranslator"
+msgstr "Bitte wählen Sie eine Rolle für die Übersetzer"
+
+msgid "plugins.generic.citationStyleLanguage.settings.citationOptionChooseChapterAuthor"
+msgstr "Bitte wählen Sie eine Rolle für die Kapitelautoren"
+
```

Comment:
We don't modify any locales except `en_US` when committing code like this. Instead, after this is merged, you'll need to provide the `de_DE` translations through [weblate](https://translate.pkp.sfu.ca/), our translation tool.